src/main/java/org/troyargonauts/Robot.java

```diff
@@ -5,11 +5,12 @@
 
 package org.troyargonauts;
 
+import org.troyargonauts.subsystems.Manipulator;
+
 import edu.wpi.first.wpilibj.TimedRobot;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.CommandScheduler;
-
-import java.util.concurrent.CompletableFuture;
+import sensors.ColorSensor;
```

Comment:
Remove Color Sensor from Robot, encapsulate in Manipulator, as it is not a subsystem